app/views/purchases/_purchase_row.html.erb

```diff
@@ -3,7 +3,7 @@
   <td><%= purchase_row.storage_location.name %></td>
   <td class="numeric"><%= purchase_row.line_items.total %></td>
   <td class="numeric"><%= purchase_row.line_items.size %></td>
-  <td class="numeric"><%= purchase_row.amount_spent_in_cents %></td>
+  <td class="numeric"><%= item_value(purchase_row.amount_spent_in_cents) %></td>
```

Comment:
This isn't a blocker, but I or @armahillo or someone should change the name of `item_value` to something like `dollar_value`

Reply:
I can do that also, better for it to be on another PR.

Reply:
Okay cool! Yes, definitely would like it on a separate PR.